vscode_extension/src/extension.ts

```diff
@@ -1,14 +1,16 @@
-import * as vscode from 'vscode';
-import { AacTaskProvider } from './AacTaskProvider';
+import { Disposable, ExtensionContext, tasks } from "vscode";
+import { AacLanguageServerClient } from "./AacLanguageServer";
+import { AacTaskProvider } from "./AacTaskProvider";
 
-let aacTaskProvider: vscode.Disposable | undefined;
+let aacTaskProvider: Disposable | undefined;
+let aacLspClient: AacLanguageServerClient = AacLanguageServerClient.makeLspClient();
 
-export function activate() {
-    aacTaskProvider = vscode.tasks.registerTaskProvider(AacTaskProvider.aacType, new AacTaskProvider());
+export function activate(context: ExtensionContext) {
+    aacTaskProvider = tasks.registerTaskProvider(AacTaskProvider.aacType, new AacTaskProvider());
+    aacLspClient.startLanguageServer(context);
 }
 
 export function deactivate(): void {
-    if (aacTaskProvider) {
-        aacTaskProvider.dispose();
-    }
+    aacTaskProvider?.dispose();
```

Comment:
Nice and clean, I love it. I totally forgot JS had this ability.

Reply:
Well, this is TS-specific, but I like the ability to use it.